tailwind.config.ts

```
@@ -1,8 +1,9 @@
-import type { Config } from "tailwindcss";
-import daisyui from "daisyui";
+import type { Config } from 'tailwindcss';
+import daisyui from 'daisyui';
+import lineClamp from '@tailwindcss/line-clamp';
```

ESLint

[error] 'lineClamp' is defined but never used.

(@typescript-eslint/no-unused-vars)